src/DataTypes/Serializations/SerializationLowCardinality.cpp

```diff
@@ -515,8 +515,14 @@ void SerializationLowCardinality::deserializeBinaryBulkWithMultipleStreams(
     size_t limit,
     DeserializeBinaryBulkSettings & settings,
     DeserializeBinaryBulkStatePtr & state,
-    SubstreamsCache * /* cache */) const
+    SubstreamsCache * cache) const
 {
+    if (auto cached_column = getFromSubstreamsCache(cache, settings.path))
```

Comment:
Could you please add some comments here? It's not easy to see where the bug is at first glace.